fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -487,6 +489,15 @@ abstract class BaseBrowserFragment :
             tabId = customTabSessionId,
         )
 
+        val copyDownloadFeature = CopyDownloadFeature(
+            context = context.applicationContext,
+            httpClient = context.components.core.client,
+            store = store,
+            tabId = customTabSessionId,
+            snackbarParent = binding.dynamicSnackbarContainer,
+            snackbarDelegate = FenixSnackbarDelegate(view),
```

Comment:
```suggestion
            snackbarDelegate = FenixSnackbarDelegate(binding.dynamicSnackbarContainer),
```

to ensure the snackbar is shown at the indended position.



https://user-images.githubusercontent.com/11428869/222115107-714f7310-5e73-418d-bb9a-874f97eac179.mp4